cypress/e2e/signIn.cy.js

```diff
@@ -12,23 +17,54 @@ describe('Sign In page', () => {
 
   before(() => {
     cy.task('db:clear');
+  });
+
+  beforeEach(() => {
     cy.task('generateUser').then((generateUser) => {
       user = generateUser;
+      signInPage.visit();
+      cy.registerUser(user.email, user.username, user.password);
```

Comment:
`registerUser` method is not defined in this context. Make sure you import it or define it in the current file.

Reply:
But it works. What I have to do? Ok, I wrote `import './commands';`